exchanges/binance/binance_wrapper.go

```diff
@@ -1882,3 +1882,15 @@ func (b *Binance) formatUSDTMarginedFuturesPair(p currency.Pair, pairFmt currenc
 	}
 	return p.Format(currency.UnderscoreDelimiter, pairFmt.Uppercase)
 }
+
+// GetServerTime returns the current exchange server time.
+func (b *Binance) GetServerTime(ctx context.Context, ai asset.Item) (time.Time, error) {
+	if !ai.IsValid() {
+		return time.Time{}, fmt.Errorf("%s %w", ai, asset.ErrNotSupported)
+	}
+
+	if ai == asset.USDTMarginedFutures {
+		return b.UServerTime(ctx)
+	}
+	return time.Time{}, common.ErrNotYetImplemented
```

Comment:
Server time is returned with:
Spot:
`GetExchangeInfo(...)`
CoinMarginedFutures:
`FuturesExchangeInfo(...)`